install/intro.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- EN-Revision: e443388d4249fea383d6ec2d1a338be220cf95ee Maintainer: lucalzetta Status: ready -->
+<!-- $Revision$ -->
+  <chapter xml:id="install.general" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink">
+   <title>Considerazioni generali sull'installazione</title>
+   <para>
+    Prima di iniziare l'installazione è necessario sapere che uso si farà 
+    di PHP. Ci sono tre settori principali in cui PHP trova applicazione 
+    come descritto in 
+    <link linkend="intro-whatcando">cosa può fare PHP?</link>
+    section:
+    <itemizedlist>
+     <listitem><simpara>Siti web e applicazioni web (programmi lato server)</simpara></listitem>
+     <listitem><simpara>Scripting da riga di comando</simpara></listitem>
+     <listitem><simpara>Applicazioni desktop (GUI)</simpara></listitem>
+    </itemizedlist>
+   </para>
+   <para>
+    Per la prima e più comune forma, sono necessarie tre cose: 
+    PHP, un server web e un browser web, probabilmente avrete già un 
+    web browser, e, conformemente alle impostazioni del vostro 
+    sistema operativo, potreste avere anche un server web (es. 
+    Apache su Linux e macOS o IIS su Windows). Potete anche aver 
+    noleggiato dello spazio web da un fornitore. In questo modo 
+    non avrete bisogno di impostare niente da parte vostra, solo 
+    scrivere i vostri script PHP, caricarli sul server che avete 
+    affittato e vedere i risultati sul vostro browser. 
+   </para>
+   <para>
+    Nel caso in cui vogliate impostare il server e PHP da voi stessi, 
+    avete due possibilità di scelta per il modo in cui collegare PHP 
+    al vostro server. Per molti server PHP ha un'interfaccia modulare 
+    diretta (altrimenti chiamata SAPI). Tra questi server sono inclusi 
+    Apache, Microsoft Internet Information Server, Netscape e iPlanet. 
+    Se PHP non ha i moduli per supportare il vostro web server è sempre 
+    possibile usarlo come processore CGI o fast CGI. Questo significa 
+    che è possibile impostare il prprio server per usare il codice 
+    eseguibile CGI di PHP per processare tutte le richieste PHP di file 
+    fatte al server. 
```

Comment:
Questa sezione ha un rigo in meno rispetto alla versione inglese. Andrebbe sistemato.